CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessSearchProviderNGTest.java

```diff
@@ -0,0 +1,327 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.dataaccess.panes;
+
+import au.gov.asd.tac.constellation.utilities.color.ConstellationColor;
+import au.gov.asd.tac.constellation.utilities.icon.UserInterfaceIconProvider;
+import au.gov.asd.tac.constellation.views.dataaccess.DataAccessUtilities;
+import au.gov.asd.tac.constellation.views.dataaccess.panes.DataAccessSearchProvider.PluginDisplayer;
+import javafx.scene.control.Tab;
+import javax.swing.Icon;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.validateMockitoUsage;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import org.mockito.MockitoAnnotations;
+import org.netbeans.spi.quicksearch.SearchRequest;
+import org.netbeans.spi.quicksearch.SearchResponse;
+import org.openide.awt.Notification;
+import org.openide.awt.NotificationDisplayer;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author aldebaran30701
+ */
+public class DataAccessSearchProviderNGTest {
+
+    @Mock
+    SearchResponse response;
+
+    @Mock
+    SearchRequest request;
+
+    @Mock
+    DataAccessPane daPane;
+
+    @Mock
+    QueryPhasePane qpPane;
+
+    protected NotificationDisplayer notifDisplayer;
+
+    private static final Icon WARNING_ICON = UserInterfaceIconProvider.WARNING.buildIcon(16, ConstellationColor.DARK_ORANGE.getJavaColor());
+
+    public DataAccessSearchProviderNGTest() {
+    }
+
+    @BeforeClass
+    public void setUpClass() throws Exception {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+    }
+
+    @BeforeMethod
+    public void setUpMethod() throws Exception {
+    }
+
+    @AfterMethod
+    public void tearDownMethod() throws Exception {
+        // Check usage is correct before moving on
+        validateMockitoUsage();
+    }
+
+    /**
+     * Test of evaluate method, of class DataAccessSearchProvider. Pass in an
+     * unmatchable search string
+     */
+    @Test
+    public void testEvaluateFail() {
+        System.out.println("evaluate fail");
+
+        // Creating mocks
+        MockitoAnnotations.openMocks(this);
+        daPane = mock(DataAccessPane.class);
+        qpPane = mock(QueryPhasePane.class);
+        request = mock(SearchRequest.class);
+        response = mock(SearchResponse.class);
```

Comment:
The `@Mock` annotation with `openMocks()` call is equivalent to calling the `mock()` on all the mock object so you'll only need to do one or the other (similar in other tests)